src/component/User/User.css

```diff
@@ -0,0 +1,17 @@
+.author__info {
```

Comment:
where is `author` class name?
if there is no, you can't add children to nonexisting parent